apps/README.md

```diff
@@ -0,0 +1,106 @@
+# Apps
+
+This directory contains all individual applications that make up studio. All applications are, for now, [Next.js](https://nextjs.org/) apps with some enhancements to make development easier.
+
+## Running an app
+
+To run an app in development mode run `dotenv next dev apps/<app>` from the _root_ of the studio repo.
+
+## Creating a new app
+
+There are several requirements to creating a new application. These will eventually be automated.
+
+1. Create a directory of the application's name in `apps`
+2. Add a `babel.config.js` to the application's root with the following contents:
+
+```js
+module.exports = require("../../babel.config");
+```
+
+3. Add a `tsconfig.json` with the following contents
+
+```json
+{
+  "extends": "../../config/tsconfig.app.json",
+  "include": ["src/**/*.tsx", "src/**/*.ts"],
+  "exclude": ["node_modules"],
+  "references": [
+    { "path": "../../libs/next" }
+  ]
+  "compilerOptions": {
+    "allowJs": true,
+    "resolveJsonModule": true,
+    "rootDir": "src",
+    "outDir": ".ts-out"
+  }
+}
+```
+
+If you want to use any other libraries aside from `@artsy-studio/next` from `libs` you _must_ add a reference field for that library under `references`. This requirement will eventually be automated.
+
+4. Create a `src` directory with a `pages` directory inside of it.
+5. Inside of `src/pages` create a `_document.tsx` file with the following contents
+
+```typescript
+export { Document as default } from "@artsy-studio/next";
+```
+
+6. Inside of `src/pages` create a `_app.tsx` file with the following contents
+
+```typescript
+export { App as default } from "@artsy-studio/next";
+```
```

Comment:
Mind adding a sentence or two here about what these exports do just so the reasoning is clear for furture devs?